final_weather_concatentation_script.ipynb

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.                    combined_df.fillna(-999, inplace = True)
</pre><p>add comment explaining why we're filling the null columns with this value</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WAT-ai/wildfire/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>